### PR TITLE
feat(VImg): use intrinsic size

### LINKED ISSUE #5757: [Bug Report] v-img does not work in v-flex with certain v-layout props
### Versions and Environment
**Vuetify:** 1.3.10
**Vue:** 2.5.17
**Browsers:** Chrome 70.0.3538.110
**OS:** Windows 10

### Steps to reproduce
Have v-layout align-center justify-center column, with flex's inside, and a v-img inside the flex. As can be seen in the codepen, the v-img only works if there is text or another component/element contained within the flex. The red border is around the flex in which the lone v-img should be appearing.

### Expected Behavior
The images should always work, obviously. It should not matter whether there are other elements in the component or flex.

### Actual Behavior
The image only appears if there is another component within the flex.

### Reproduction Link
<a href="https://codepen.io/tkted/live/BGqJEK" target="_blank">https://codepen.io/tkted/live/BGqJEK</a>

### LINKED ISSUE #17031: [Bug Report][3.1.12] Vimg inside VBtn does not render
### Environment
**Vuetify Version:** 3.1.12
**Last working version:** 2.6.14
**Vue Version:** 3.2.47
**Browsers:** Edge 111.0.1661.54
**OS:** Windows 10

### Steps to reproduce
Adding a v-img element inside a v-btn (either in the default slot, or in the prepend slot) does not render anything.

### Expected Behavior
Expected v-img to be rendered and visible

### Actual Behavior
v-img is not rendered

### Reproduction Link
[https://play.vuetifyjs.com/#...](https://play.vuetifyjs.com/#eNrFU01vGjEQ/SuDe1gi2HVDW1WigOit/6CHbA5m1ywGf8k2hAjx3zv2Llsgag+5RIqUffPm470Z83QiP60tDntOpmQWuLKSBb4oNcDskDNr02cCigndoYQDP4Z8LbisAUlTczkvifJNSYBe5wnVwFTUSGbqFUGGCYod8w0XzSZgePKti7yIOmwuAe8q/NyEYP2U0qrWUWQQ69etLyqjqFCs4Z5a5piU7EgV6naCyWJrUcKNglXQkATgB3ZeVlJUO4TpP68xdGBYqqOY6KokfTXWX5YCn6zjluv6igToYryGVNkzZfh46yie3py0C3rL9CKZB8UHM5pwvy+aFna5O6L+8BG0T+KmL0JfOWEDeB72kRbKGhfgBI6v4QxrZxRk6CGLyZXRPgA+FJhHfpj94lIa+G2crAfZw49LRqVDl/H5KtieDInhA8wXcIq6ImUkL6Rphv1VByWJZRAnFQcm9xxGc8Dt4t8oNm+DKaVHoxHic7JEW0/ohoxJawg3YYutNxp/K2lw2RG+JNNWSoyh0Yj/XnCv7a5Jl1siR91eB6F4Xhu1/FJMiq/faS18uI4X3Kt85cyL5w4nlmR81Ty+hH8N6Gjs+1g8Ttq+XSyXbOVj4zcNKU45cJe7+JAdd/9Vf5d74+COe+MizsTtnnGjUuidv1tm5dMiny7S3mEvtYj1z3HMeUxaeTiwLSHPfwDcN5x6)

### PR DESCRIPTION
**Pull request by KaelWD**
## Description

closes #17031
closes #5757 (again)

## Markup:

```vue
<template>
  <v-app>
    <v-app-bar color="secondary">
      <v-img inline src="https://d22f547j6rcbz1.cloudfront.net/images/onlinedraft-50y.png" />
      <v-img inline src="https://d22f547j6rcbz1.cloudfront.net/images/onlinedraft-black.png" />
    </v-app-bar>
    <v-main>
      <v-container>
        <v-card>
          <v-responsive :aspect-ratio="16/9">
            Lorem ipsum dolor sit amet consectetur adipisicing elit. Commodi, ratione debitis quis est labore voluptatibus! Eaque cupiditate minima, at placeat totam, magni doloremque veniam neque porro libero rerum unde voluptatem!

            <br>

            <v-img src="https://d22f547j6rcbz1.cloudfront.net/images/onlinedraft-50y.png">
              Lorem ipsum dolor sit amet consectetur adipisicing elit. Commodi, ratione debitis quis est labore voluptatibus! Eaque cupiditate minima, at placeat totam, magni doloremque veniam neque porro libero rerum unde voluptatem!
            </v-img>
          </v-responsive>
        </v-card>

        <v-btn variant="text">
          <template #prepend>
            prepended text
            <v-img max-height="25" max-width="25" src="https://cdn.vuetifyjs.com/images/parallax/material.jpg" />
          </template>
          <span>click me!</span>
        </v-btn>
      </v-container>
    </v-main>
  </v-app>
</template>
```

### CODE REVIEW

File: packages/vuetify/src/components/VImg/VImg.tsx [hunk 1]
```diff
@@ -288,33 +286,36 @@ export const VImg = genericComponent<VImgSlots>()({
...
+    useRender(() => {
+      const [responsiveProps] = VResponsive.filterProps(props)
+      return (
+        <VResponsive
+          class={[
+            'v-img',
+            { 'v-img--booting': !isBooted.value },
+          ]}
+          { ...responsiveProps }
+          aspectRatio={ aspectRatio.value }
```
Comment (johnleider) [inline packages/vuetify/src/components/VImg/VImg.tsx:298]: does this need to be defined? it's handled in responsive props

Comment (KaelWD) [reply packages/vuetify/src/components/VImg/VImg.tsx:298]: yes

**Review by johnleider — DISMISSED** [review]

**Comment by johnleider** [comment]
The image snaps in on refresh. First part of the gif is using **img** and the second part is using **v-img**:

image-flash

**Comment by kappaflow** [comment]
@KaelWD Why did you decide to do a such breaking change in a minor update?

I have `VImg` with `max-height="fit-content" max-width="fit-content"` in many places. And now such images have zero size and basically disappeared:
[Example](https://play.vuetifyjs.com/#eNqNUctOwzAQ/JXI5zoWj1MVEEj8BeHgJtvEwXYs7yYtQv131imlIQi1t2RndmZ2/PopnkPIxwHEWhQELlhN8Fj6LCtqM2aV1YgPpajl1sI+6wYks/2QGHQFUsd+8HWmrWm8rMATxGzTyCbCh+RZS+DlfSkmtYWe0zNkgRHsSeKwIUMW5C3TXmCrB0uFYtp5Z5TGNae/7GedE+xaQ1CKM+b0XraQIjFha0hWPaf1tOTsTE3t/xSMFYMtUcC1UlXtU3GpkA7zqnfKON0AqqCjtlbvleMuo9E270Jz1nks1BT91MvPVaevQs0eQqyENf4d8w57z4/0mZilqBBLsc5ejxq/Q3VYgzVjzD2Q8sGp75RPd/ldfnPLLkinmbR6w+GTWpJ6K/2BHY0LfSTpdFjYHoHJepqkGSul/wsZnpim4uDJOJB176Yw98css3EO6OQm9juEyCKlWM1sUuArrC6eyx5/tBUbjhBlBF9DhHjtSYu1+VkL6M9pyf2QCj+sxLTGzR8zi7cvJ9gxJA==)

And now I am not even sure what is the right way to fix it, while preserving the same functionality I had.

**Pull request merged**